pkg/sfu/downtrack.go

```diff
@@ -1675,6 +1711,52 @@ func (d *DownTrack) HandleRTCPSenderReportData(_payloadType webrtc.PayloadType,
 	return nil
 }
 
+func (d *DownTrack) SetUpstreamRTT(rttMs uint32) {
+	if d.upstreamRtt.Load() == rttMs {
+		return
+	}
+
+	d.upstreamRtt.Store(rttMs)
+	d.updatePlayoutDelay()
+}
+
+func (d *DownTrack) updatePlayoutDelay() {
+	if d.playoutDelayExtID == 0 {
+		return
+	}
+
+	rttMax := d.upstreamRtt.Load()
+	if rtt := d.rtpStats.GetRtt(); rtt > rttMax {
+		rttMax = rtt
+	}
+
+	// target delay is 30% more than RTT
+	targetDelay := rttMax * 13 / 10
+	currentDelay := d.playoutDelay.Load()
+	if currentDelay != nil {
+		currentTargetDelay := uint32(currentDelay.(rtpextension.PlayOutDelay).Min) * 10
```

Comment:
Pushing the factor of 10 into the play out delay extension code, we can avoid all the mysterious divide by 10, multiply by 10 in code.